modules/nf-core/ribowaltz/meta.yml

```diff
@@ -0,0 +1,97 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "ribowaltz"
+description: Calculation of optimal P-site offsets, diagnostic analysis and visual inspection of ribosome profiling data.
+keywords:
+  - sort
+  - example
+  - genomics
+tools:
+  - "ribowaltz":
+      description: "Calculation of optimal P-site offsets, diagnostic analysis and visual inspection of ribosome profiling data."
+      homepage: "https://github.com/LabTranslationalArchitectomics/riboWaltz"
+      documentation: "https://github.com/LabTranslationalArchitectomics/riboWaltz"
+      tool_dev_url: "https://github.com/LabTranslationalArchitectomics/riboWaltz"
+      doi: "10.1371/journal.pcbi.1006169"
+      licence: ["MIT"]
+
+input:
+  # Only when we have meta
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'sample1', single_end:false ]`
+  - meta2:
+      type: map
+      description: |
+        Map containing reference information for the reference genome GTF file
+        e.g. `[ id:'Ensembl human v.111' ]`
+  - meta3:
+      type: map
+      description: |
+        Map containing reference information for the reference genome FASTA file
+        e.g. `[ id:'Ensembl human v.111' ]`
+
+  - bam:
+      type: file
+      description: Transcriptome BAM file
+      pattern: "*.{bam}"
+  - gtf:
+      type: file
+      description: GTF file of reference genome
+      pattern: "*.{gtf.gz,gtf}"
+  - fasta:
+      type: file
+      description: FASTA file of reference genome
+      pattern: "*.{fasta,fa,fasta.gz,fa.gz}"
+
+output:
+  #Only when we have meta
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'sample1', single_end:false ]`
+
+  - versions:
```

Comment:
Can you order these the same as in the main.nf for ease of comparison?

Reply:
done, see commit 2c16f8e.